serial_constants.go

```diff
@@ -0,0 +1,26 @@
+package serial
+
+type DataBitsType byte
+type StopBitsType byte
+type ParityType byte
+
+const (
+	DATABITS_8 DataBitsType = iota
+	DATABITS_7
```

Comment:
It would be nice to make DATABITS_7 = 7, etc.

Obviously 0 should be handled specially and get mapped to 8.